### PR TITLE
Sync with remote

### PR DESCRIPTION
**Pull request by carlos-m159**
Updating branch with remote.

### CODE REVIEW

**Comment by carlos-m159** [comment]
I am not sure how to proceed with the change log and versions. How should we proceed here?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -41,6 +41,81 @@ Changelog for package urg_node
 * Tailor: Creating Jenkinsfile
 * Contributors: locus-services
 
+0.1.18 (2022-03-03)
```
Comment (svwilliams) [inline CHANGELOG.rst:44]: I _think_ this is correct. The upstream versions are all grouped together, and the Locus version history is added on top of that. But we'll let @garyservin tell us for sure.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
```
Comment (svwilliams) [inline .travis.yml]: I'm not sure what this file does, or if we want it as part of our fork. I've added @garyservin as a reviewer so he can advise on these buildserver questions.

**Review by svwilliams — APPROVED** [review]

File: .travis.yml (hunk 2 above)
Comment (garyservin) [reply .travis.yml]: we would need to remove this file to make sure we don't start a travis test for this.

File: CHANGELOG.rst (hunk 1 above)
Comment (garyservin) [reply CHANGELOG.rst:44]: yeah, it should be fine

**Review by garyservin — APPROVED** [review]
LGTM

File: .travis.yml (hunk 2 above)
Comment (carlos-m159) [reply .travis.yml]: Removing it!

**Pull request merged**